protos/Tokens/SBTokenHarness.proto

```diff
@@ -0,0 +1,62 @@
+#VRML_OBJ R2021b utf8
+# tags: static
```

Comment:
Could use a comment explaining why this harness is needed

Reply:
Added in 8bce23d72ef92f47a2c1a275b8c5f0b1793d2433